qcfractal/testing.py

```diff
@@ -372,13 +382,18 @@ def build_managed_compute_server(mtype):
         from qcfractal.queue import QueueManager
         manager = QueueManager(client, adapter_client)
 
-        # Yield the server instance
         yield client, server, manager
 
         # Close down and clean the adapter
         manager.close_adapter()
 
 
+@pytest.fixture(scope="module", params=["pool", "dask", "fireworks", "parsl"])
+def adapter_client_fixture(request):
+    adapter_client = build_adapter_clients(request.param)
+    yield adapter_client
```

Comment:
Maybe add an explicit close after the yield?

Reply:
Back to this comment as you are having issues. Good to do an explicit close here.